guides/source/action_view_overview.md

```diff
@@ -517,43 +556,50 @@ CAUTION: Only keyword arguments are supported. Defining positional or block argu
 Layouts
```

Comment:
Not sure how you're thinking about structuring this vs the layouts and rendering guide, but it might make sense to consider having at least a basic example of how an application layout looks like, for the sake of giving an introduction/overview to it here.

Reply:
Good idea. Added an example with a simple layout and explain that the view content is rendered in place of the `<%= yield %>`